### PR TITLE
[feature] execution run-mode support streaming and batch

### LINKED ISSUE #1642: [Bug] The main method caused an error: StreamTableEnvironment can not run in batch mode for now, please use TableEnvironment.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

After I changed the mode of the built-in sqldemo from streaming to batch, I reported an error

### StreamPark Version

1.2.4

### Java Version

_No response_

### Flink Version

1.13.5

### Scala Version of Flink

2.12

### Error Exception

```log
After I changed the mode of the built-in sqldemo from streaming to batch, I reported an error
```

### Screenshots

f41f711301c0b3462875617213041de
ba019eaf5598ec5b1f51e7642021f50

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wolfboys**
execution run-mode support streaming and batch

### What problem does this PR solve?

Issue Number: close #1642 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/streampark/issues).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ProjectServiceImpl.java [hunk 1]
```diff
@@ -178,35 +177,36 @@ public void build(Long id, String socketId) throws Exception {
         StringBuilder builder = new StringBuilder();
         tailBuffer.put(id, builder.append(project.getLog4BuildStart()));
         boolean cloneSuccess = cloneSourceCode(project, socketId);
-        if (!cloneSuccess) {
+        if (cloneSuccess) {
```
Comment (1996fanrui) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ProjectServiceImpl.java]: This file change maybe a misoperation?

Comment (wolfboys) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ProjectServiceImpl.java]: > This file change maybe a misoperation?

Thanks for your feedback, done✌️

**Review by 1996fanrui — APPROVED** [review]
LGTM

**Review by xinzhuxiansheng — APPROVED** [review]
LGTM

**Pull request merged**